.github/workflows/docker-integration-test.yaml

```diff
@@ -0,0 +1,31 @@
+name: docker-integration-test
+on: [push]
+jobs:
+  docker-integration-test:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
The github actions are mainly lift and shift but I updated the versions to the latest.